products/models.py

```diff
@@ -0,0 +1,65 @@
+from django.db import models
+
+class Category(models.Model):
+    name = models.CharField(max_length=45)
+
+    class Meta:
+        db_table = 'categories'
+
+class Collection(models.Model):
+    name = models.CharField(max_length=45)
+
+    class Meta:
+        db_table = 'collections'
+
+class Gender(models.Model):
+    name = models.CharField(max_length=45)
+
+    class Meta:
+        db_table = 'genders'
+
+class Product(models.Model):
+    product_name = models.CharField(max_length=45)
+    price        = models.IntegerField()
+    genders      = models.ForeignKey(Gender, on_delete=models.CASCADE, related_name="gender_products")
+    is_adult     = models.BooleanField(default=True)
+    collections  = models.ForeignKey(Collection, on_delete=models.CASCADE, related_name="collection_products")
+    categories   = models.ForeignKey(Category, on_delete=models.CASCADE, related_name="category_products")
+
+    class Meta:
+        db_table = 'products'
+
+class Color(models.Model):
+    name = models.CharField(max_length=45)
+
+    class Meta:
+        db_table = 'colors'
+
+class Product_Color(models.Model):
+    products = models.ForeignKey(Product, on_delete=models.CASCADE, related_name="product_products_colors")
```

Comment:
`products`라고 작성하신다면 `products_id`가 됩니다. FK로 하나의 데이터만 참조하는 경우는 단수로 작성해주세요.
```suggestion
    product = models.ForeignKey(Product, on_delete=models.CASCADE, related_name="product_products_colors")
```